### ISSUE TITLE
while running app getting following error

### ISSUE THREAD

**Issue by kumar0608** [github/saml-ldap-team-sync#44]
Traceback (most recent call last):
  File "app.py", line 11, in <module>
    from githubapp import GitHubApp, LDAPClient, TEST_MODE, CRON_INTERVAL
  File "/Users/sravula/Downloads/saml-ldap-team-sync-2.0/githubapp/__init__.py", line 27, in <module>
    TEST_MODE = strtobool(os.environ.get('TEST_MODE', False))
  File "/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/util.py", line 301, in strtobool
    val = val.lower()
AttributeError: 'bool' object has no attribute 'lower'
OSXSRAVUMBP15:saml-ldap-team-sync-2.0 sravula$

**Comment by primetheus** [comment]
@kumar0608 can you try changing https://github.com/github/saml-ldap-team-sync/blob/master/githubapp/__init__.py#L27 to put quotes around `False` and see if that fixes it?

```python
TEST_MODE = strtobool(os.environ.get('TEST_MODE', 'False'))
```

**Comment by primetheus** [comment]
I have a PR opened for this in #45

**Comment by kumar0608** [comment]
HI Jared,

sorry to bother you again, can you please advise about this error?

File
"/Users/sravula/Downloads/ldap2/saml-ldap-team-sync/githubapp/core.py",
line 36, in load_env
    app.config['GITHUBAPP_ID'] = int(os.environ['APP_ID'])

already i have this value in the .env file

**Comment by kumar0608** [comment]
HI Jarad,

we are getting the above error

what is the purpose of these below env variables?

WEBHOOK_SECRET=development
## GitHub App ID
APP_ID=12345
## Private Key Path
PRIVATE_KEY_PATH=.ssh/team-sync.pem

**Comment by primetheus** [comment]
@kumar0608 can you tell me how you installed the requirements for this, and how you are running it?

**Comment by primetheus** [comment]
the variables will be filled in with the data from creating the GitHub App... You can find more information about how to create a GitHub App here: https://developer.github.com/apps/building-github-apps/creating-a-github-app/

**Comment by kumar0608** [comment]
yeah i configured app in github enterprise and provided APP details in the
.env file

Thanks
Sridhar

**Comment by kumar0608** [comment]
if you have 5 min time, will you be able to help on this?

Thanks
Sfidhar

**Comment by primetheus** [comment]
I'm closing this one, as the original issue has been solved in #45. The present conversation revolves around #46
